INSTALL.md

```diff
@@ -23,7 +23,17 @@ For Debian-like:
 
 For Fedora-like:
 
-- dnf install gcc cmake make fuse-devel polarssl-devel ruby-devel
+- dnf install gcc cmake make fuse-devel mbedtls-devel ruby-devel rubypick
+
+Alternatively, running "dnf install dislocker fuse-dislocker" to use the
+already existing RPM packages in Fedora could be a clever idea.
+
+For RHEL-like (including CentOS Scientific Linux):
+
+- yum install gcc cmake make fuse-devel mbedtls-devel ruby-devel ruby
```

Comment:
Do we need 'ruby' here too? Isn't ruby-devel already sufficient, and will pull ruby?